CMakeLists.txt

```diff
@@ -75,18 +75,31 @@ find_package(PythonInterp)
 find_package(PythonLibs)
 set (BUILD_PYTHON ${PYTHONLIBS_FOUND})
 if (BUILD_PYTHON)
-  set(PYTHON_DEST "${CMAKE_INSTALL_PREFIX}/python")
+  set(PYTHON_INSTALL_DIR "" CACHE PATH "Directory of the SIRF Python modules")
```

Comment:
I seem to recall we chose `PYTHON_DEST_DIR` to avoid confusion with "my files" vs "system files"?

Reply:
damn you're right! https://github.com/CCPPETMR/SIRF/pull/160#issuecomment-384928229